### ISSUE TITLE
Desktop Library | Clicking Webelement errors with object has no attribute 'startswith'

### ISSUE THREAD

**Issue by amcanirn** [Accruent/robotframework-zoomba#211]
Using the 'Get Webelement' example from the Desktop Library documentation results in an error.

AttributeError: 'WebElement' object has no attribute 'startswith'

robotframework-zoomba version: 2.11.0

```
*** Settings ***
Library           Zoomba.DesktopLibrary
Suite Setup       Start App

*** Variables ***
${REMOTE_URL}           http://127.0.0.1:4723
${APP}                  Microsoft.WindowsCalculator_8wekyb3d8bbwe!App

*** Keywords ***
Start App
    Open Application    ${REMOTE_URL}     platformName=Windows    deviceName=Windows   app=${APP}    alias=Main

*** Test Cases ***
Click Webelement
    ${element}    Get Webelement    name=One
    Click Element    ${element}
```

![image](https://user-images.githubusercontent.com/51360049/123343978-c5565e00-d54a-11eb-9630-5bb80c14faa3.png)

**Comment by Wolfe1** [comment]
Adding some unit tests around it but it should be fixed in that linked pull request. Should have this merged after I have a member of my team review this morning.

**Comment by Wolfe1** [comment]
@amcanirn Released in 2.11.1: https://github.com/Accruent/robotframework-zoomba/releases/tag/2.11.1

Update should be available via pip in a few minutes. Let me know if this addresses your issue fully.

**Comment by amcanirn** [comment]
Yes this looks to have fixed the issue for me. Thank you for sorting that so quickly!
